packages/home-spa/src/styles/main.scss

```diff
@@ -29,13 +29,13 @@
         q {
             position: relative;
             &::before {
-                content: url('/img/begin-quote.svg');
+                content: url('../img/begin-quote.svg');
                 position: relative;
                 top: -18px;
                 left: -8px;
             }
             &::after {
-                content: url('/img/end-quote.svg');
+                content: url('../img/end-quote.svg');
```

Comment:
Again, do we really need a separate image here? Simply using translate should suffice.

Reply:
Translate isn't supported in firefox yet https://caniuse.com/?search=translate @sayak-sarkar